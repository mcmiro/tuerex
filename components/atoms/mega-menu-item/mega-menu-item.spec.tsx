import * as React from "react";
import {screen, render} from "@testing-library/react";
import MegaMenuItem from "./";

describe("Test MegaMenuItem component", () => {
	it("Should pass when link is defined", () => {
		const linkMock = {
			title: "test",
			url: "/test",
		};

		render(<MegaMenuItem link={linkMock} data-testid="test" />);
		const megaMenu = screen.queryByTestId("test");
		expect(megaMenu).toBeDefined();
		expect(megaMenu).toMatchSnapshot();
	});
});
