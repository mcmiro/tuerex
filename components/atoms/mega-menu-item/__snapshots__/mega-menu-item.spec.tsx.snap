// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MegaMenuItem component Should pass when link is defined 1`] = `null`;
